string parent_frame
string child_frame
float64 x
float64 y
float64 theta
---
bool indicator